dashboard/app/helpers/levels_helper.rb

```diff
@@ -23,19 +23,35 @@ def build_script_level_path(script_level, params = {})
       flappy_chapter_path(script_level.chapter, params)
     elsif params[:puzzle_page]
       if script_level.lesson.numbered_lesson?
-        puzzle_page_script_stage_script_level_path(script_level.script, script_level.lesson, script_level, params[:puzzle_page])
+        puzzle_page_script_lesson_script_level_path(script_level.script, script_level.lesson, script_level, params[:puzzle_page])
       else
         puzzle_page_script_lockable_stage_script_level_path(script_level.script, script_level.lesson, script_level, params[:puzzle_page])
       end
     elsif params[:sublevel_position]
-      sublevel_script_stage_script_level_path(script_level.script, script_level.lesson, script_level, params[:sublevel_position])
+      sublevel_script_lesson_script_level_path(script_level.script, script_level.lesson, script_level, params[:sublevel_position])
     elsif !script_level.lesson.numbered_lesson?
       script_lockable_stage_script_level_path(script_level.script, script_level.lesson, script_level, params)
     elsif script_level.bonus
       query_params = params.merge(level_name: script_level.level.name)
-      script_stage_extras_path(script_level.script.name, script_level.lesson.relative_position, query_params)
+      script_lesson_extras_path(script_level.script.name, script_level.lesson.relative_position, query_params)
     else
-      script_stage_script_level_path(script_level.script, script_level.lesson, script_level, params)
+      script_lesson_script_level_path(script_level.script, script_level.lesson, script_level, params)
+    end
+  end
+
+  # This is a a temporary method to help with moving translations onto the new level urls. To start this will
+  # keep translations on the old URL until foundations can do the work to bring them over to the new url
+  def build_script_level_path_for_translations(script_level)
```

Comment:
I noticed this doesn't have the exact same code as the old `build_script_level_path`. Is that intentional?

Reply:
Yes it is. I noticed that the translations call to build_script_level_path was not using the second paramater `param` and that means it would never hit certain cases. So I just kept the cases that it would hit